app/policies/portfolio_policy.rb

```diff
@@ -27,11 +27,19 @@ def update?
 
   class Scope < Scope
     def resolve
-      if catalog_administrator?
+      access_scopes = @user_context.access.scopes(scope.table_name, 'read')
```

Comment:
@eclarizio Shouldn't this be able to use the Scope from the parent class.

Reply:
I can change that, sure.